airflow/providers/qubole/operators/qubole.py

```diff
@@ -39,7 +41,13 @@ class QDSLink(BaseOperatorLink):
 
     name = 'Go to QDS'
 
-    def get_link(self, operator: BaseOperator, dttm: datetime) -> str:
+    def get_link(
+        self,
+        operator: "AbstractOperator",
+        dttm: Optional[datetime] = None,
+        *,
+        ti_key: Optional["TaskInstanceKey"] = None,
+    ) -> str:
```

Comment:
I think this signature and logic should be backwards and forwards compatible.

Reply:
Oh not quite -- I can't use the "AbstractOperator" type as that only exists in 2.3